benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -25,13 +25,23 @@ void GraphicsBenchmarkApp::InitKnobs()
     PPX_ASSERT_MSG(!cl_options.HasExtraOption("vs-shader-index"), "--vs-shader-index flag has been replaced, instead use --vs and specify the name of the vertex shader");
     PPX_ASSERT_MSG(!cl_options.HasExtraOption("ps-shader-index"), "--ps-shader-index flag has been replaced, instead use --ps and specify the name of the pixel shader");
 
+    pEnableSpheres = GetKnobManager().CreateKnob<ppx::KnobCheckbox>("enable-spheres", true);
+    pEnableSpheres->SetDisplayName("Enable Spheres");
+    pEnableSpheres->SetFlagDescription("Enable the Spheres in the scene.");
+
+    pEnableSkyBox = GetKnobManager().CreateKnob<ppx::KnobCheckbox>("enable-skybox", true);
+    pEnableSkyBox->SetDisplayName("Enable SkyBox");
+    pEnableSkyBox->SetFlagDescription("Enable the SkyBox in the scene.");
+
     pKnobVs = GetKnobManager().CreateKnob<ppx::KnobDropdown<std::string>>("vs", 0, kAvailableVsShaders);
     pKnobVs->SetDisplayName("Vertex Shader");
     pKnobVs->SetFlagDescription("Select the vertex shader for the graphics pipeline.");
+    pKnobVs->SetIndent(1);
 
     pKnobPs = GetKnobManager().CreateKnob<ppx::KnobDropdown<std::string>>("ps", 0, kAvailablePsShaders);
     pKnobPs->SetDisplayName("Pixel Shader");
     pKnobPs->SetFlagDescription("Select the pixel shader for the graphics pipeline.");
+    pKnobPs->SetIndent(1);
 
     pAllTexturesTo1x1 = GetKnobManager().CreateKnob<ppx::KnobCheckbox>("all-textures-to-1x1", false);
```

Comment:
Should be indent of 2 (sorry, accidentally deleted this comment earlier)

Reply:
Done.